stream_alert/rule_processor/config.py

```diff
@@ -100,21 +102,18 @@ def load_env(context):
         context: The AWS Lambda context object.
 
     Returns:
-        {'lambda_region': 'region_name',
-         'account_id': <ACCOUNT_ID>,
-         'lambda_function_name': 'function_name',
-         'lambda_alias': 'qualifier'}
+        [dict] including the following items:
```

Comment:
super nit: should be `(dict)` per [this](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#type-annotations)